cranelift/filetests/filetests/isa/x64/load-op-store.clif

```diff
@@ -0,0 +1,147 @@
+test compile precise-output
+target x86_64
+
+function %f0(i64, i32) {
+block0(v0: i64, v1: i32):
+  v2 = load.i32 v0+32
+  v3 = iadd v2, v1
+  store v3, v0+32
+  return
+}
+
+;   pushq   %rbp
+;   movq    %rsp, %rbp
+; block0:
+;   addl    32(%rdi), %esi, 32(%rdi)
```

Comment:
I don't think these should be presented in 3-operand form. It doesn't match x86_64, which is confusing.

Reply:
Yeah, that's probably true; I was trying to mirror the SSA-ified view of the reg-reg forms but with a memory dest I think it's just as clear in "actual" form (since the memory dest is a source from a regalloc perspective). Updated.